src/object/srv_ec_aggregate.c

```diff
@@ -1587,23 +1601,33 @@ agg_process_holes(struct ec_agg_entry *entry)
 		rc = dss_abterr2der(rc);
 		goto ev_out;
 	}
-
+#if HUH
```

Comment:
hmm, just curious, what is this?

Reply:
That is not supposed to be there.  Sydney took out the lines of code that in the if block when she did the Coverity fixes, and I wanted to verify that it was okay.  Sorry.  I took it out now!